src/action.cpp

```diff
@@ -1980,6 +1980,16 @@ void actionUpdateDroid(DROID *psDroid)
 		break;
 	case DACTION_MOVETODROIDREPAIR:
 		{
+			BASE_OBJECT* actionTargetObj = psDroid->psActionTarget[0];
+			ASSERT_OR_RETURN(, actionTargetObj->type == OBJ_DROID, "unexpected repair target");
```

Comment:
While this should not be `nullptr`, I'd feel happier with a null check here beforehand (given issues we've had elsewhere).
```suggestion
			ASSERT_OR_RETURN(, actionTargetObj != nullptr && actionTargetObj->type == OBJ_DROID, "unexpected repair target");
```